index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
```

Comment:
- [ ] [REQUIRED] I think you forgot to import the `style.css` file into the `HTML` file. Therefore, the style given to the text in the header tag does not work. Please implement my suggestion 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/211609618-4d9ae0e9-7076-4a3c-9b5c-1531db7f7cea.png)